Add workflow definition for fireperf end-to-end tests

yifanyang opened the pull request.

yifanyang commented:
@vkryachko A successful execution of this workflow can be found at https://github.com/firebase/firebase-android-sdk/actions/runs/2800178050.

vkryachko reviewed (commented):
LGTM overall.

It's unclear to me how notifications will work though, [afaict](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs) you will be the one getting failure notifications, not the fireperf team. wdyt?

yifanyang commented:
> LGTM overall.
> 
> It's unclear to me how notifications will work though, [afaict](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs) you will be the one getting failure notifications, not the fireperf team. wdyt?

Added the last step that create/update issue to notify fireperf folks. Here is an example issue #3977. 

I have removed the `pull_request` trigger, therefore the tests did not run for last commit. Successful runs can be found at: 
- https://github.com/firebase/firebase-android-sdk/runs/7757358619?check_suite_focus=true
- https://github.com/firebase/firebase-android-sdk/runs/7757358696?check_suite_focus=true

vkryachko commented on .github/workflows/fireperf-e2e.yml:
```diff
@@ -0,0 +1,107 @@
+                title: 'FirePerf E2E Test Failures',
+                body: text,
+                labels: ['fireperf-e2e-tests'],
+                assignees: ['jeremyjiang-dev', 'leotianlizhan', 'visumickey']
```
fyi @jeremyjiang-dev @leotianlizhan @visumickey does that look good to you?

jeremyjiang-dev replied:
Could you also add @raymondlam ? LGTM

yifanyang replied:
Done.
